orcid-commons/src/main/java/no/sikt/nva/orcid/commons/service/OrcidServiceImpl.java

```diff
@@ -28,16 +32,30 @@ public OrcidServiceImpl(String orcidTableName, AmazonDynamoDB client, Clock cloc
 
     @Override
     public OrcidCredentials createOrcidCredentials(OrcidCredentials orcidCredentials) {
+
+        var orcidCredentialsWithTimeStamp = createOrcidCredentialsWithTimeStamp(orcidCredentials);
+        return insertOrcidCredentials(orcidCredentialsWithTimeStamp);
+    }
+
+    private OrcidCredentials createOrcidCredentialsWithTimeStamp(OrcidCredentials orcidCredentials) {
         var currentTime = clockForTimestamps.instant();
-        orcidCredentials.setModified(currentTime);
-        orcidCredentials.setCreated(currentTime);
-        return insertOrcidCredentials(orcidCredentials);
```

Comment:
Not setters in a record! 

(You can make a weird workaround inside the record, but anyone who does that has lost the point of records IMO)